src/plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -61,19 +65,44 @@ module.exports['generateReviewAssignments'] = async function (input: any, DBConn
           isLastLevel: nextReviewLevel === numReviewLevels,
         }
     })
+    console.log('ReviewAssignments', reviewAssignments)
```

Comment:
I think this one is for dev ?

Reply:
I think it's ok to log reviewAssignmentIds returned by `addReviewAssignmentAssignerJoins`, though

Reply:
Okay, I'll just comment out the first one.